FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,34 @@
+#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int8.h>
+#include <Arduino.h>
+std_msgs::Int8 fuel_level_percent_full;
+ros::Publisher pub_fuel_level("fuel_level_percent_full", &fuel_level_percent_full);
+ros::NodeHandle nh;
+
+const long LOOP_DELAY = 200;
+const int READ_PIN = A7;
```

Comment:
```suggestion
const int FUEL_SENSOR_PIN = A7;
```

Reply:
This actually has a good name now, thanks